### PR TITLE
Trending view

### PR DESCRIPTION
**Pull request by mattprecious**
Let me know where pixels should be pushed.

device-2015-03-16-182719

### CODE REVIEW

File: src/main/java/com/jakewharton/u2020/ui/misc/DividerItemDecoration.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2014 The Android Open Source Project
```
Comment (mattprecious) [inline src/main/java/com/jakewharton/u2020/ui/misc/DividerItemDecoration.java]: Most of this is copied from AOSP, however padding and RTL related changes are my own.

**Comment by JakeWharton** [comment]
Padding on divider should be even on both sides. I'll look more closely
later.

File: src/main/java/com/jakewharton/u2020/ui/trending/TrendingAdapter.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  @Override public long getItemId(int position) {
+    return position;
+  }
+
+  @Override public int getItemCount() {
+    return repositories.size();
+  }
+
+  public static final class ViewHolder extends RecyclerView.ViewHolder {
```
Comment (JakeWharton) [inline src/main/java/com/jakewharton/u2020/ui/trending/TrendingAdapter.java]: I was going to say delete this and just use the view... but oh yeah.. RecyclerView.

File: src/main/java/com/jakewharton/u2020/ui/trending/TrendingItemView.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+  public TrendingItemView(Context context, AttributeSet attrs) {
+    super(context, attrs);
+
+    TypedValue outValue = new TypedValue();
+    context.getTheme().resolveAttribute(android.R.attr.textColorSecondary, outValue, true);
+    descriptionColor = outValue.data;
+
+    avatarTransformation =
+        new CircleStrokeTransformation(context, getResources().getColor(R.color.avatar_stroke), 1);
```
Comment (JakeWharton) [inline src/main/java/com/jakewharton/u2020/ui/trending/TrendingItemView.java]: Add a TODO to make this a singleton or at least share it across all views in this adapter.

File: src/main/java/com/jakewharton/u2020/ui/trending/TrendingView.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+import com.jakewharton.u2020.util.ViewSubscriptions;
+import com.squareup.picasso.Picasso;
+import javax.inject.Inject;
+import org.joda.time.DateTime;
+
+import static com.jakewharton.u2020.ui.misc.DividerItemDecoration.VERTICAL_LIST;
+
+public final class TrendingView extends BetterViewAnimator {
+  // Item start padding (16) + avatar width (40) + avatar margin (16).
+  private static final int DIVIDER_PADDING_START_DP = 72;
```
Comment (JakeWharton) [inline src/main/java/com/jakewharton/u2020/ui/trending/TrendingView.java]: ya dimens this. soon you'll be able to `@ResourceDimen` it!

**Comment by JakeWharton** [comment]
lgtm

**Comment by JakeWharton** [comment]
lGtm

File: src/main/java/com/jakewharton/u2020/ui/trending/TrendingView.java (hunk 4 above)
Comment (mattprecious) [inline src/main/java/com/jakewharton/u2020/ui/trending/TrendingView.java]: Want.

**Pull request merged**